tests/stdlib/tlists.nim

```diff
@@ -81,3 +81,63 @@ block tlistsToString:
     l.append('2')
     l.append('3')
     doAssert $l == """['1', '2', '3']"""
+
+template testCommon(initList, toList) =
+
+  block: # toSinglyLinkedList, toDoublyLinkedList
+    let l = seq[int].default
+    doAssert l.toList.toSeq == []
+    doAssert [1].toList.toSeq == [1]
+    doAssert [1, 2, 3].toList.toSeq == [1, 2, 3]
+
+  block copy:
```

Comment:
this test doesn't actually test correctness of `copy`.

(Remember that runnableExamples is only run for 1 specific backend, unlike this which will be tested on all backends (eventually, once https://github.com/nim-lang/Nim/issues/16384 is fixed))